File: CPP/Kode/PetShop.cpp
#include <iostream>
#include <iomanip>
using namespace std;

class Petshop {
protected:
    string id, nama, brand;
    double harga;
    int stok;
public:
    Petshop(string id="", string nama="", string brand="", double harga=0, int stok=0)
        : id(id), nama(nama), brand(brand), harga(harga), stok(stok) {}

    string getId() { return id; }
    string getNama() { return nama; }
    string getBrand() { return brand; }
    double getHarga() { return harga; }
    int getStok() { return stok; }

    virtual void tampilkanData() const {
        cout << "| " << setw(6) << id << " | " << setw(18) << nama << " | " << setw(13) << brand 
             << " | " << setw(8) << harga << " | " << setw(4) << stok << " | ";
    }
    
};

File: PHP/Kode/Petshop.php
<?php
class petshop {
    private $id;
    private $nama_produk;
    private $harga_produk;
    private $stok_produk;
    private $foto_produk;

    public function __construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk) {
        $this->id = $id;
        $this->nama_produk = $nama_produk;
        $this->harga_produk = $harga_produk;
        $this->stok_produk = $stok_produk;
        $this->foto_produk = $foto_produk;
    }

    public function getData() {
        return [
            'id' => $this->id,
            'nama' => $this->nama_produk,
            'harga' => $this->harga_produk,
            'stok' => $this->stok_produk,
            'foto' => $this->foto_produk
        ];
    }
}
?>